### PR TITLE
document monkeypatch_session

### PR DESCRIPTION
**Pull request by graingert**
In a slightly safer way

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -47,6 +47,32 @@ Does not provide:
 - SFTP. Security is for the weak.
 - Less common commands.
 
+Monkey Patching
+---------------
+
+Sometimes you may want to call a library that uses requests with an ftp URL.
+First ensure that the library does not take a session parameter:
```
Comment (Lukasa) [inline README.rst]: Let's change this: "First, check whether the library takes a session parameter. If it does, you can use the `FTPSession` object, which is the preferred approach."

Comment (graingert) [inline README.rst]: First, check whether the library takes a session parameter. If it does, you can use the FTPSession class (or FTPAdaptor), which is the preferred approach

Comment (Lukasa) [inline README.rst]: Works for me.

Comment (graingert) [inline README.rst]: or:

First, check whether the library takes a session parameter. If it does, you can use either the `FTPSession` class or `FTPAdaptor` directly, which is the preferred approach

File: README.rst [hunk 2]
```diff
@@ -47,6 +47,34 @@ Does not provide:
 - SFTP. Security is for the weak.
 - Less common commands.
 
+Monkey Patching
+---------------
+
+Sometimes you may want to call a library that uses requests with an ftp URL.
+First, check whether the library takes a session parameter. If it does, you
+can use either the FTPSessios or FTPAdapter class directly, which is the preferred
```
Comment (Lukasa) [inline README.rst]: Typo here, you want `Session`.

**Pull request merged**